diff --git a/.eslintrc.local.js b/.eslintrc.local.js
index 7f62918..e36675d 100644
--- a/.eslintrc.local.js
+++ b/.eslintrc.local.js
@@ -15,5 +15,19 @@ module.exports = {
         strict: ['error', 'global'],
       },
     },
+    {
+      files: ['benchmarks/bench-compare.js', 'benchmarks/bench-diff.js'],
+      rules: {
+        'import/no-extraneous-dependencies': [
+          'error',
+          {
+            devDependencies: true,
+          },
+        ],
+        'import/no-nodejs-modules': ['error'],
+        'no-console': ['error'],
+        strict: ['error', 'global'],
+      },
+    },
   ],
 }
diff --git a/benchmarks/bench-compare.js b/benchmarks/bench-compare.js
index ec75f2d..8725358 100644
--- a/benchmarks/bench-compare.js
+++ b/benchmarks/bench-compare.js
@@ -15,6 +15,6 @@ for (const [v0, v1] of comparisons) {
 
 suite
   .on('cycle', function (event) {
-    console.log(String(event.target))
+    process.stdout.write(`${String(event.target)}\n`)
   })
   .run({ async: false })
diff --git a/benchmarks/bench-diff.js b/benchmarks/bench-diff.js
index c447fb3..f5ceced 100644
--- a/benchmarks/bench-diff.js
+++ b/benchmarks/bench-diff.js
@@ -18,6 +18,6 @@ for (const [v1, v2] of cases) {
 
 suite
   .on('cycle', function (event) {
-    console.log(String(event.target))
+    process.stdout.write(`${String(event.target)}\n`)
   })
   .run({ async: false })
